textile.go

```diff
@@ -162,6 +172,58 @@ func (x *commandsCmd) Execute(args []string) error {
 	return nil
 }
 
+func (x *docsCmd) Execute(args []string) error {
```

Comment:
New command `docs` generates a giant markdown file with all the commands usages text. Sub commands are further indented.